lisa/tools/qemu.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy.assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix, Ubuntu
+from lisa.tools import Aria2c, Lscpu, Lsmod
+
+
+class Qemu(Tool):
+
+    NESTED_VM_IMAGE_NAME = "image.qcow2"
+
+    @property
+    def command(self) -> str:
+        return "qemu-system-x86_64"
+
+    def can_install(self) -> bool:
+        return True
+
+    def _install(self) -> bool:
+        assert isinstance(self.node.os, Posix)
+
+        if isinstance(self.node.os, Ubuntu):
+            # verify that virtualization is enabled in hardware
+            is_virtualization_enabled = self.node.tools[
+                Lscpu
+            ].is_virtualization_enabled()
+            assert_that(
+                is_virtualization_enabled, "Virtualization is disabled"
+            ).is_true()
+
+            # install qemu
+            self.node.os.install_packages("bridge-utils")
+            self.node.os.install_packages("dnsmasq")
+            self.node.os.install_packages("qemu-kvm")
+
+            # verify that kvm is enabled
+            self._is_kvm_successfully_enabled()
+        else:
+            raise NotImplementedError(f"{self.node.os} is not supported")
+
+        return self._check_exists()
+
+    def _is_kvm_successfully_enabled(self) -> None:
+        lsmod_output = self.node.tools[Lsmod].run().stdout
+        is_kvm_successfully_enabled = "kvm_intel" in lsmod_output
```

Comment:
we need consider not only intel but also other architecture.

Reply:
Thanks Lili! I will modify the code to include amd based information as well.